src/scripts/notify.sh

```diff
@@ -111,11 +106,29 @@ BranchFilter() {
     fi
 }
 
+CheckEnvVars() {
+    if [ -z "$SLACK_ACCESS_TOKEN" ]; then
+        echo "In order to use the Slack Orb (v4 +), an OAuth token must be present via the SLACK_ACCESS_TOKEN environment variable."
+        echo "Follow the setup guide available in the wiki: https://github.com/CircleCI-Public/slack-orb/wiki/Setup"
+        exit 1
+    fi
+    # If no channel is provided, quit with error
+    if [ -z "$SLACK_PARAM_CHANNEL" ]; then
+       echo "No channel was provided. Enter value for SLACK_DEFAULT_CHANNEL env var, or channel parameter"
+       exit 1
+    fi
+    if [ -n "$SLACK_WEBHOOK" ]; then
+        echo "It appears you have a Slack Webhook token present in this job."
+        echo "Please note, Webhooks are no longer used for the Slack Orb (v4 +)."
+        echo "Follow the setup guide available in the wiki: https://github.com/CircleCI-Public/slack-orb/wiki/Setup"
+    fi
+}
+
 # Will not run if sourced from another script.
 # This is done so this script may be tested.
 ORB_TEST_ENV="bats-core"
 if [ "${0#*$ORB_TEST_ENV}" = "$0" ]; then
-    . "/tmp/SLACK_JOB_STATUS"
```

Comment:
CC @gmemstr I think this may be from the old orb as the blame shows you added this, but it was during the major re-write. Im testing now removing this. Do you have any idea what this might have been? Looks to be sourcing a tmp file that I wouldnt expect to exist. I think it was possibly used in the old orb, not sure how I didnt see it.

Reply:
I don't recall removing/adding `. "/tmp/SLACK_JOB_STATUS"`, so I don't really know :(

Reply:
Looks like this has been referenced for carrying over the job's status and removing this introduced a bug.

I see this output, but not sure where it comes from:

*Slack - Detecting Job Status (FAIL)*
```
echo 'export CCI_STATUS="fail"' > /tmp/SLACK_JOB_STATUS
```

Specifying `event: fail` stopped working for us.

Reply:
Ahh it was hidden in the other command / run step of course 🤦 very sorry about that. Patching this now

Reply:
Thank you @skyeagle